### PR TITLE
Check for appropriate keystore file type on upload

### PR DESCRIPTION
**Pull request by aitrean**
Closes https://github.com/MyEtherWallet/MyEtherWallet/issues/590

### CODE REVIEW

File: common/components/WalletDecrypt/Keystore.tsx [hunk 1]
```diff
@@ -18,11 +19,17 @@ function isPassRequired(file: string): boolean {
   return passReq;
 }
 
+function isValidFile(rawFile: any): boolean {
```
[the remark this answers is not in the document]
Comment (HenryNguyen5) [reply common/components/WalletDecrypt/Keystore.tsx]: Is it not part of the `FileReader` interface?

**Comment by dternyak** [comment]
@aitrean Are you planning on updating the `any` type based on @HenryNguyen5's suggestion?

**Comment by aitrean** [comment]
Apologies, meant to add that and it slipped by me. One moment.

**Comment by dternyak** [comment]
Thank you for the contribution @aitrean! :)

**Pull request merged**